### PR TITLE
Increased text size of "Select the form to open" #1374

### PR DESCRIPTION
**Pull request by laddha-adi**
The header text for the form list in ODK Survey "Select the form to open" was quite small and hard to notice. I have increased its text size from 14 sp to 21 sp.
closes #1374

### CODE REVIEW

**Comment by himanshusagar** [comment]
Hey @laddha-adi ,

I was thinking about this issue.  Since you've already opened an pull request and changing "Select the form to open" text's size will scales entire UI, I was wondering whether putting dimension value in values-ldpi , values-mdpi etc would make sense? Maybe one of admins can enlighten us on this ?

**Comment by laddha-adi** [comment]
@opendatakit-bot  closes#1374

**Comment by jbeorse** [comment]
runtests

File: survey_app/build.gradle [hunk 1]
```diff
@@ -114,10 +114,9 @@ dependencies {
     implementation 'com.android.support:support-annotations:27.0.2'
     implementation 'com.android.support:support-v13:27.0.2'
     implementation 'com.google.firebase:firebase-core:11.6.2'
-    implementation ('com.crashlytics.sdk.android:crashlytics:2.8.0@aar') {
-        transitive =true;
+    implementation('com.crashlytics.sdk.android:crashlytics:2.8.0@aar') {
+        transitive = true;
```
Comment (jbeorse) [inline survey_app/build.gradle]: Please remove whitespace changes like this and like lines 120, 131, etc. Please isolate the actual code changes.

Comment (laddha-adi) [reply survey_app/build.gradle]: Sure, would be doing the required changes soon.

**Comment by jbeorse** [comment]
You have made your changes on the "master" branch. We actually make all new changes on the "development" branch and then merge them downstream into "demo" and "master" as the build becomes stable and we make releases. Please move these changes into the "development" branch and make a pull request against that. I don't think this should be too difficult as the files haven't changed much.

The actual text size change is good. I can merge when the above changes are complete.

**Comment by jbeorse** [comment]
Its ok, don't worry. You're not the first to make that mistake. When your new pull request is in I will merge it.

**Pull request closed without merging**